lib/chore_runner/reporter.ex

```diff
@@ -40,8 +40,8 @@ defmodule ChoreRunner.Reporter do
   def report_failed(reason),
     do: GenServer.cast(get_reporter_pid(), {:chore_failed, reason, DateTime.utc_now()})
 
-  def log(message),
-    do: GenServer.cast(get_reporter_pid(), {:log, message, DateTime.utc_now()})
+  def log(message, opts \\ %{html: false, timestamp: true}),
```

Comment:
Can you re-write this to use a keyword list of options? Just to be consistent with the rest of the library